pom.xml

```diff
@@ -122,6 +122,11 @@
         <plugin>
           <artifactId>maven-compiler-plugin</artifactId>
           <version>3.8.1</version>
+          <configuration>
+          	<fork>true</fork>
+          	<useIncrementalCompilation>true</useIncrementalCompilation>
+
+          </configuration>\n
```

Comment:
I see the characters "\n" here. Don't know how that happened?
Also there is an unnecessary empty line above that could be removed.